public/assets/js/login.js

```diff
@@ -6,12 +6,21 @@ let buttonEnter = document.getElementById("js-button-loggin");
 //checkUser();
 
 buttonEnter.addEventListener("click", function (event) {
-  event.preventDefault();
-  let usernameValue = $username.value;
-  if (usernameValue) {
-    console.log(usernameValue);
-    //guardar username en local storage
-  } else {
-    console.log("vacio");
-  }
+    event.preventDefault();
+    let usernameValue = $username.value;
+    if (usernameValue) {
+        console.log(usernameValue);
+        //guardar username en local storage
+        almacenarUser(usernameValue)
```

Comment:
Aquí podríamos redirigir al main
```suggestion
        almacenarUser(usernameValue)
        window.location = 'main.html';
```